extensions/sql-bindings/package.json

```diff
@@ -41,7 +41,7 @@
         "command": "sqlBindings.createAzureFunction",
         "title": "%sqlBindings.createAzureFunction%",
         "category": "MS SQL",
-        "when": "view == objectExplorer && viewItem == Table"
+        "when": "view == objectExplorer && viewItem == Table && !azdataAvailable"
```

Comment:
Is this when even valid here? I don't see "when" being a valid clause for "commands" contributions 

https://github.com/microsoft/vscode/blob/main/src/vs/workbench/services/actions/common/menusExtensionPoint.ts#L527

Are you seeing this correctly hidden in ADS or something?